src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './NewCommentForm.scss';
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+export const NewCommentForm = ({ addComment }) => {
+  const [commentName, setCommentName] = useState('');
+  const [commentEmail, setCommentEmail] = useState('');
+  const [commentBody, setCommentBody] = useState('');
+  const [validate, setValidate] = useState(true);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    if (!commentName.trim() || !commentEmail.trim() || !commentBody.trim()) {
```

Comment:
```suggestion
    if ([commentName, commentEmail, commentBody].some(item => !item.trim())) {
```
you could also use such construction